engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/errata-content-hosts.controller.js

```diff
@@ -64,7 +64,9 @@ angular.module('Bastion.errata').controller('ErrataContentHostsController',
         };
 
         $scope.goToNextStep = function () {
+            $scope.$parent.numberOfContentHostsToUpdate = nutupane.table.resource.total;
```

Comment:
this wouldn't handle if you unselected any right?

I think you want to call nutupane.table.allResultsSelectCount()

That handles the subtotal (the search), plus subtracting unselected items.